app/routables/index/styles.scss

```diff
@@ -1,168 +1,307 @@
-main#homepage.page {
-  section#nav {
-    height: 0;
-    nav {
+#homepage {
+  #hero {
+    background-image: url("/assets/images/weekend-warrior-hero.jpg");
+    height: 100vh;
+    position: relative;
+    background-size: cover;
+    background-position-x: 80%;
+    h1 {
       position: absolute;
-      background-color: #1c1c1c;
-      opacity: 0.9;
-      display: flex;
-      align-items: center;
-      justify-content: space-between;
-      padding: 0px 60px;
-      z-index: 1;
-      width: 100%;
-      box-sizing: border-box;
-      .left,
-      .right {
-        display: flex;
-      }
-      .left {
-        a {
-          font-family: $regulargirassol;
-          font-size: 30px;
+      bottom: 0;
+      text-align: right;
+      width: 92%;
+      padding: 57px;
+    }
+    #menu-wrap {
+      postion: fixed;
+      top: 20px;
+      right: 80px;
+      .toggler:checked ~ .menu {
```

Comment:
Quite nifty to use a checkbox to toggle hamburger menu. What does the `~` do?

Reply:
It is general sibling combinator and similar to Adjacent sibling combinator. the difference is that the second selector does NOT have to immediately follow the first one means It will select all elements that is preceded by the former selector.